### PR TITLE
Add STIX2 output capability to multiscanner

### PR DESCRIPTION
**Pull request by emmanvg**
This PR adds additional functionality by processing contents of a submission report and creating STIX2 Indicators from them. It currently generates content from:

- Cuckoo Sandbox output
- General submission information

There is still room to improve the STIX content generation.

### CODE REVIEW

**Comment by awest1339** [comment]
@emmanvg can we add unit tests for this functionality please?

File: utils/api.py [hunk 1]
```diff
@@ -898,6 +900,35 @@ def get_pdf_report(task_id):
...
+@app.route('/api/v1/tasks/<task_id>/stix2', methods=['GET'])
+def get_stix2_bundle_from_report(task_id):
+    '''
+    Generates a STIX2 Bundle with indicators generated of a JSON report.
+    '''
+    formatting = request.args.get('pretty', default='False', type=str)[0].lower()
+
+    report_dict, success = get_report_dict(task_id)
+
+    if formatting == 't' or formatting == 'y' or formatting == '1':
```
Comment (clenk) [inline utils/api.py]: You could move the formatting stuff below line 918, so if get_report_dict is not successful, it returns more quickly.

**Comment by emmanvg** [comment]
@clenk is my target `mitre:feature_celery` still correct?

**Comment by awest1339** [comment]
I like the idea of using feature_celery or develop as a staging branch, then merging to master and doing a release.
Along these lines: http://datasift.github.io/gitflow/IntroducingGitFlow.html
@Drewsif, thoughts?

**Comment by clenk** [comment]
I think we should use `dev` as the staging/development branch and delete `feature_celery` once the documentation PR is merged in, since Celery is in master now. Use the branch that's named what you're using it for.

**Pull request merged**